lms/djangoapps/api_manager/courses/tests.py

```diff
@@ -2599,3 +2602,31 @@ def test_course_navigation(self):
             },
             response.data
         )
+
+    def test_course_aggregate_metadata_update_on_course_published(self):
```

Comment:
Course aggregate data is used in course completions metrics API which route through api_manager app.